lib/src/easy_localization_app.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:easy_localization/easy_localization.dart';
```

Comment:
I don't think this file should import the package itself, I feel like that can introduce circular dependency issues. Not sure how Dart deals with it, though.

Reply:
Not sure either about circular dep issues but there is no need to import the package here, so i removed it. Good catch 👍